diracx-logic/src/diracx/logic/jobs/status.py

```diff
@@ -39,6 +42,15 @@
 
 logger = logging.getLogger(__name__)
 
+SQL_HEARTBEAT_FIELDS = [
+    "LoadAverage",
+    "MemoryUsed",
+    "Vsize",
+    "AvailableDiskSpace",
+    "CPUConsumed",
+    "WallClockTime",
+]
+
```

Comment:
would it be better to pull this out of the Table object instead, or from a list defined in `db.py` that pulls this out from the Table object?